core/src/test/scala/pickling/run/anypickler-unpickler.scala

```diff
@@ -0,0 +1,60 @@
+package scala.pickling.runtime
+
+import org.scalatest.FunSuite
+import scala.pickling._
+
+import scala.pickling.json.JsonFormats
+import scala.pickling.pickler.AllPicklers
+import scala.pickling.json.JSONPickle
+import scala.pickling.internal.HybridRuntime
+
+abstract class PicklingLogic extends Ops with AllPicklers with JsonFormats
+
+object PicklingProtocol extends {
+  val ignoreMe = internal.replaceRuntime(new HybridRuntime)
+} with PicklingLogic {
+  implicit val so = static.StaticOnly
+}
+
+case class Box[T](value: T)
+
+class AnyPicklerUnpickler extends FunSuite {
+
+  import PicklingProtocol._
+
+  test("AnyPickler should work") {
+
+    val b = Box(List(1,2,3,4,5))
+    val containerPickler = implicitly[Pickler[List[Int]]]
+    val pickled = b.pickle.value
```

Comment:
Note, you'll need to actually register the pickler with the hybrid runtime (currently) so this is expected to fail.

Specifically:

```scala
scala.pickling.internal.currentRuntime.picklerRegistry.registerPickler(containerPickler)
```

Then, AnyPickler should be able to work with hybrid mode.